cost/azure/rightsize_managed_disks/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v2.0.1
+
+- Fixed error related to the calculation of unitOfMeasure for the method findUltraPrice for ultra sized disk
```

Comment:
- Fixed bug that caused wrong calculations for ultra tier disks

Reply:
Done